transformer_engine/pytorch/csrc/userbuffers/userbuffers-host.cpp

```diff
@@ -13,12 +13,11 @@
 #include <sched.h>
 #include <stdio.h>
 #include <string.h>
-#include <transformer_engine/userbuffers.h>
-#include <transformer_engine/logging.h>
```

Comment:
Why not use logging.h?

Reply:
This way userbuffers can be a standalone lib not dependent on libtransformerengine in any way